### PR TITLE
Server-side rendering of topic events

### PR DESCRIPTION
**Pull request by julianlam**
- style: reformat list of helpers exported
- refactor: move topic events partial into a helper, invoke helper in topic.tpl (see persona), update how events are added to the DOM via addTopicEvents

### CODE REVIEW

**Comment by julianlam** [comment]
# Important

* [ ] Merge the one-liner change in Persona/Vanilla too

**Comment by julianlam** [comment]
# :rotating_light: Breaking change

Themes that modify `topic.tpl` will need to add the following line

```
<ul component="topic" class="posts timeline" data-tid="{tid}" data-cid="{cid}">
	{{{each posts}}}
		<li ...>
			... snip snip...
		</li>
		{renderTopicEvents(@index)}      <--- Add this line
	{{{end}}}
</ul>
```

**Comment by julianlam** [comment]
See the changesets above, for how this breaking change was implemented in vanilla and Persona, respectively :point_up:

File: public/src/modules/helpers.js [hunk 1]
```diff
@@ -208,6 +210,46 @@
...
+				if (!event.user.system) {
+					html += `<span><a href="${relative_path}/user/${event.user.userslug}">${buildAvatar(event.user, 'xs', true)}&nbsp;${event.user.username}</a></span>&nbsp;`;
+				} else {
+					html += `<span class="timeline-text">[[global:system-user]]</span>&nbsp;`;
+				}
+			}
+
+			html += `<span class="timeago timeline-text" title="${event.timestampISO}"></span>`;
+
+			if (this.privileges.isAdminOrMod) {
```
Comment (barisusakli) [inline public/src/modules/helpers.js:245]: this.privileges might be undefined here

**Pull request merged**